html/about-us.html

```diff
@@ -0,0 +1,233 @@
+                            <div class="card">
+                                <div class="card__side card__side--front">
+                                    <div class="card__picture card__picture--india-3">
+                                        &nbsp;
+                                    </div>
+                                    <h4 class="card__heading">
+                                        <span class="card__heading-span card__heading-span--india">
+                                            Ranjitha
+                                        </span>
+                                    </h4>
+                                    <div class="card__details">
+                                        <ul>
+                                            <li>India</li>
+                                            <li>Front-End</li>
+                                            <li>Futuregames</li>
+                                        </ul>
+                                    </div>
+                                </div>
+
+                                <div class="card__side card__side--back card--india">
+                                    <div class="card__flag--india">
+
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="grid__item">
+                            <div class="card">
+                                <div class="card__side card__side--front">
+                                    <div class="card__picture card__picture--uganda">
+                                    </div>
+                                    <h4 class="card__heading">
+                                        <span class="card__heading-span card__heading-span--uganda">
+                                            Andrew
+                                        </span>
+                                    </h4>
+                                    <div class="card__details">
+                                        <ul>
+                                            <li>Uganda</li>
+                                            <li>Front-End</li>
+                                            <li>Futuregames</li>
+                                        </ul>
+                                    </div>
+                                </div>
+
+                                <div class="card__side card__side--back card--uganda">
+                                    <div class="card__flag--uganda"></div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="grid__item">
+                            <div class="card">
+                                <div class="card__side card__side--front">
+                                    <div class="card__picture card__picture--sweden">
+                                        &nbsp;
```

Comment:
can you explain this

Reply:
&nbsp; is a html typo for generate a "space". i used it to just give some info on the card picture section to trigger events.